packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -749,6 +757,9 @@ export async function launch({
             type: 'cosmic-swingset-bootstrap-block-start',
             blockTime,
           });
+          // Start a block transaction, but without changing state
+          // for the upcoming begin block check
+          saveBeginHeight(0);
```

Comment:
Why "without changing state"?  Saving a 0 is a state change, isn't it?

Reply:
The default is `0` when missing, so writing `0` instead of defaulting does not effectively change the state. I'll calrify.

Reply:
I could either re-write it as `saveBeginHeight(savedBeginHeight);` or update the comment, or both. Which one would be more clear?